docs/src/configuration/routes/_index.md

```diff
@@ -64,31 +64,77 @@ The route for the `api` app could be anything in the domain, even a subdomain li
 Be aware that using a subdomain might [double your network traffic](https://nickolinger.com/blog/2021-08-04-you-dont-need-that-cors-request/),
 so consider using a path like `https://{default}/api` instead.
 
-## Route templates
+## Route placeholders
 
-Each route in your configuration file is defined in one of two ways:
+Each route in your `.platform/routes.yaml` configuration file is defined in one of two ways:
 
-* An absolute URL such as `https://example.com/blog`
-* A URL template such as `https://{default}/blog`
+* An explicit URL such as `https://example.com/blog`
+* A placeholder URL such as `https://{default}/blog`
+
+Placeholders can be either `{default}` or `{all}`, and they map to the [custom domains](../../domains/quick-start.md) that 
+you have defined in your project settings.
 
-The templates include a placeholder, either `{default}` or `{all}`,
-to stand in for [custom domains](../../domains/quick-start.md) you've defined in your project.
 These domains can be top level domains (`example.com`) or subdomains (`app.example.com`).
 
 ### `{default}`
 
 `{default}` represents your default custom domain.
-If you have set your default domain to `example.com`,
-`example.com` and `{default}` in your `.platform/routes.yaml` file have the same meaning for your production environment.
 
-Each development environment gets its own domain with a unique identifier.
-If you use the URL template on a `feature` branch, you get something like the following:
+Assuming that you set your default domain to `example.com`.
+
+Using both `example.com` and `{default}` in your `.platform/routes.yaml` file generates the same URLs for your **production** environment.
+
+However, your **development** environment URLs differ depending on the method you use.
+
+1. If you use the default placeholder:
+
+```yaml {location=".platform/routes.yaml"}
+"https://{default}/":
+    type: upstream
+    upstream: "app:http"
+```
+
+The generated development URL of the `feature` environment is:
 
 ```txt
 https://feature-t6dnbai-abcdef1234567.us-2.platformsh.site/blog
 ```
 
-If you use the absolute URL method (like `example.com`), that domain looks similar to this:
+2. If you use the explicit domain method:
+
+```yaml {location=".platform/routes.yaml"}
+"https://example.com/":
+    type: upstream
+    upstream: "app:http"
+```
+
+The generated development URL of the `feature` environment is:
+
+```txt
+https://example.com.feature-t6dnbai-abcdef1234567.us-2.platformsh.site/blog
+```
+
+WARNING: THIS INCONSISTENT BEHAVIOR IS FIXED IN THE UPCOMING RELEASE.
```

Comment:
```suggestion
{{< note theme="warning">}} THIS INCONSISTENT BEHAVIOR IS FIXED IN THE UPCOMING RELEASE. {{< /note >}}
```
this seem to use the "warning" theme, like here: https://docs.platform.sh/administration/web/configure-project.html#domains
I think, the closing `{{< /note >}}` could be moved further, if you want to include the text below into the warning message.

Reply:
I like the idea of a note. I think the warning context might be different, so I think it'd be good to define a pattern for it.

Reply:
Although the warning theme isn't so bad in context :thinking:

Reply:
Can see it here: https://pr-2255-hpnyxbi-652soceglkw4u.eu-3.platformsh.site/configuration/routes.html#default